docs/diagrams/DeleteVisitActivityDiagram.puml

```diff
@@ -4,7 +4,7 @@ start
 'Since the beta syntax does not support placing the condition outside the
 'diamond we place it as the true branch instead.
 
-repeat :User executes deleteVisit command;
+repeat :User executes deleteLocation command;
```

Comment:
Is this supposed to be deleteVisit instead? It is in the DeleteVisitActivityDiagram.puml file.

Reply:
It is a mistake. Fixed.